setup.cfg

```diff
@@ -30,7 +31,10 @@ project_urls =
 [options]
 packages = find:
 install_requires =
-    numpy>=1.18.5
+    numpy>=1.18.5; platform_machine!='aarch64' and platform_machine!='arm64' and python_version<'3.10'
+    numpy>=1.19.2; platform_machine=='aarch64'
+    numpy>=1.20.0; platform_machine=='arm64'
+    numpy>=1.21.0; python_version=='3.10'
```

Comment:
Is there a reason to list these out? If you are on Python 3.10, there simply won't be a version of Python older than 1.21, if you are on a special arch, same thing; so why limit it here? Also this exact string won't work when Python 3.11 comes along.